packages/description-list-react/documentation/Example.tsx

```diff
@@ -0,0 +1,25 @@
+import { ExampleComponentProps } from "@fremtind/jkl-portal-components";
+import React from "react";
+import { DescriptionList, DescriptionDetail, DescriptionTerm } from "../src";
+import "./Example.scss";
+
+export const Example: React.FC<ExampleComponentProps> = () => {
```

Comment:
Det er ikke så kritisk med eksempelkode, men for å pirke litt :smiley: : hvorfor er denne komponenten typet sånn?
Ikke er det en `FC`, for den tar ikke `children`.
Ikke trenger den `ExampleComponentProps`, den tar jo ikke props. Eller er det noe med opplegget jeg ikke kjenner til?

Reply:
Fjernet ExampleComponentProps og endret fra FC til VFC